pyproject.toml

```diff
@@ -34,7 +35,7 @@ include=[
 "Issue tracker" = "https://github.com/redis/redis-om-python/issues"
 
 [tool.poetry.dependencies]
-python = "^3.7,<=3.11"
+python = ">=3.7,<3.12"
```

Comment:
Please change this to:
python = ">=3.10,<4.0"

Or it affects default poetry install. And requires editing in the pyproject.toml file as a work around.

Reply:
This is even better.
Fixed.

Reply:
I thought there was a reason for initially capping it to Python 3.11 only.
So I just fixed the limit to include all 3.11.x variations.

Reply:
In most cases, older python code should be compatible in newer python versions.
So this makes sense. Especially 3.7+